File: ClientApp/src/types/dto/admin/extendedGameProgressDto.ts
type PlayerWithTargetDto = {
	playerId: string;
	victimId: string;
	playerFullName: string;
	victimFullName: string;
};

type ExtendedGameProgressDto = {
	alivePlayers: number;
	playerData: PlayerWithTargetDto[];
};

export default ExtendedGameProgressDto;

File: Dto/ExtendedGameProgressDto.cs
namespace Assassins.Dto;

public class ExtendedGameProgressDto
{
	public class PlayerWithTargetDto
	{
		public Guid PlayerId { get; set; }
		public Guid VictimId { get; set; }
		public string PlayerFullName { get; set; } = null!;
		public string VictimFullName { get; set; } = null!;
	}

	public int AliveUsers { get; set; }
	public List<PlayerWithTargetDto> PlayerData { get; set; } = null!;
}